### ISSUE TITLE
add an option to show combined weights in for multiclass classifiers

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#56]
Currently if there are several classes weights for them are shown separately. I think sometimes it could be helpful to show them all in the same table (or even highlight them all in text).

**Comment by lopuhin** [comment]
Yes, I think both of these features are great to have.

**Comment by lopuhin** [comment]
Now weights for multiclass classifiers are shown in a horizontal layout by default (#62), but there is nothing done for feature highlighting in text.
Some ideas about how it can be done are in https://github.com/TeamHG-Memex/eli5/pull/62#issuecomment-259149457
